scss/atoms/buttons/_buttons.scss

```diff
@@ -39,7 +39,7 @@
 }
 
 .btn {
-	@include btn(	map-get($ui-btn, margin));
+	@include btn(map-get($ui-btn, margin));
 }
 
```

Comment:
Probably shouldn't make these changes as part of this PR, since this PR is image focused